src/main/java/com/minecolonies/coremod/tileentities/TileEntityWareHouse.java

```diff
@@ -152,26 +153,27 @@ public AbstractBuilding getTask()
 
     /**
      * Check if the required items by the building are in the wareHouse.
+     *
      * @param buildingEntry the building requesting.
-     * @param addToList if is in warehouse should add to the list?
+     * @param addToList     if is in warehouse should add to the list?
      * @return true if has something in warehouse to deliver.
      */
     public boolean checkInWareHouse(@NotNull final AbstractBuilding buildingEntry, final boolean addToList)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Refactor this method to reduce its Cognitive Complexity from 26 to the 20 allowed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS3776)